src/core/qgsmaprendererjob.h

```diff
@@ -199,9 +199,18 @@ class CORE_EXPORT QgsMapRendererJob : public QObject
      */
     void setCache( QgsMapRendererCache *cache );
 
-    //! Find out how long it took to finish the job (in milliseconds)
+    /**
+     * Returns the total time it took to finish the job (in milliseconds).
+     * \see layerRenderingTime()
+     */
     int renderingTime() const { return mRenderingTime; }
 
+    /**
+     * Returns the render time (in ms) per layer, by layer ID.
+     * \since QGIS 3.0
+     */
+    QMap< QString, int > layerRenderingTime() const { return mLayerRenderingTime; }
```

Comment:
wouldn't perLayerRenderingTime() better reflect that this is not just the time for a single layer. Or layersRenderingTime() ?

Reply:
I'll change it to perLayerRenderingTime()